### PR TITLE
adding regex implementation for cleaning the commits

### PR DESCRIPTION
**Pull request by yushghimire**
- used regex to clean commits

### CODE REVIEW

File: __init__.py [hunk 1]
```diff
@@ -11,9 +11,11 @@
 separated_lines = [] # 🥥
 
 for line in reversed(lines):
-    line_without_hyphen = line.split("-") # 😆
-    separated_lines.append(line_without_hyphen[1])
+    lines_without_hyphen = re.sub('^[\w\d]*\s(.)', '', line)
```
Comment (sbimochan) [inline __init__.py:15]: Where's the import of re?

Comment (yushghimire) [reply __init__.py:15]: oh!!!! simple mistake.

Comment (yushghimire) [reply __init__.py:15]: addressed.

Comment (sbimochan) [reply __init__.py:15]: cool

**Pull request merged**